index.js

```diff
@@ -43,7 +43,7 @@ module.exports = class WebpackCopyModulesPlugin {
         .forEach(fileDependencies.add.bind(fileDependencies)));
 
     return Promise.all([...fileDependencies].map(function(file) {
-      const relativePath = replaceParentDirReferences(path.relative('', file)),
+      const relativePath = replaceParentDirReferences(path.relative(process.cwd(), file)),
```

Comment:
Normally, `path.relative` takes the empty string to mean the current working directory.  However, Jest mocks `process`, so the `chdir` call in the test doesn't truly change the current working directory.  That mock doesn't get picked up by `path` apparently, so it still uses the real current working directory rather than the output from the mocked `process.cwd()` call.  Therefore for testability this line has been changed to use `process.cwd()` explicitly.